lib/iris/coords.py

```diff
@@ -143,7 +143,7 @@ def __new__(cls, name_or_coord, minimum, maximum,
                                       'groupby_point, groupby_slice')
 
 
-def _discontinuity_in_2d_bounds(bds, abs_tol=1e-4):
+def _discontiguity_in_2d_bounds(bds, abs_tol=1e-4):
```

Comment:
I've changed this to be `discontiguity` for consistency as we use the word `discontiguous` everywhere else

Reply:
Agreed.  Why is it private though, is there a good reason?